pkg/util/conditions/service.go

```diff
@@ -21,16 +21,6 @@ func SetServiceCreated(conditions *[]metav1.Condition, conditionType, name strin
 	_ = meta.SetStatusCondition(conditions, condition)
 }
 
-func SetServiceUpdated(conditions *[]metav1.Condition, conditionType, name string) {
```

Comment:
`ServiceUpdatedReason` const is unused now

Reply:
fixed in cd8132dc